site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -440,7 +409,7 @@ var backgroundPageConnection = chrome.runtime.connect({
 
 ## DevTools extension examples
 
-Browse the source of these DevTools extension examples:
+Browse the source of these DevTools extension examples: <!--Does this mean that the examples on this page come from these links? Or that these links provide more examples? How is this section different from the "examples" section at the end?-->
```

Comment:
The former. We have yet to convert most of these to Manifest V3. @oliverdunk What's the priority new DevTools examples?

Reply:
Updated. Thanks!

Reply:
> @oliverdunk What's the priority new DevTools examples?

My recollection is that most of our MV2 samples were too dated to be relevant (e.g using frameworks that have since disappeared). I think some new ones would be nice at some point but probably low priority.